### PR TITLE
Realtime Fix for gRPC and SQL

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud  
SQL takes in an integer ID. I had to add the `AcceptableIdType` function to allow integers also.

### CODE REVIEW

File: grpc_server.go [hunk 1]
```diff
@@ -373,7 +373,7 @@ func (s *server) RealTime(stream pb.SpaceCloud_RealTimeServer) error {
 
 			// Subscribe to relaitme feed
 			feedData, err := s.realtime.Subscribe(ctx, clientID, s.auth, s.crud, data, func(feed *model.FeedData) {
-				client.Write(&model.Message{Type: utils.TypeRealtimeFeed, Data: feed})
+				client.Write(&model.Message{ID: req.ID, Type: utils.TypeRealtimeFeed, Data: feed})
```
Comment (YourTechBud) [inline grpc_server.go]: Why do we need to send the id here?

**Comment by YourTechBud** [comment]
The implentation may pose this problem, the ids in the docs array will be of type string even when its an int in the database. Correct?

File: grpc_server.go (hunk 1 above)
Comment (AliabbasMerchant) [reply grpc_server.go]: Just to be extra sure that we are responding to the correct client

**Comment by AliabbasMerchant** [comment]
I think no. I got back an integer in the python client

File: grpc_server.go (hunk 1 above)
Comment (YourTechBud) [reply grpc_server.go]: Okay. I don't think that is required. The client receiving the message is identified by the `client` object we are calling write on

**Comment by YourTechBud** [comment]
Ah yes! You are right.

File: grpc_server.go (hunk 1 above)
Comment (AliabbasMerchant) [reply grpc_server.go]: I am checking for the id at the client side. Serves as a double check. Like how we check during the Subscribe response. As it is, we have the id field empty. Lets use it. 😜

**Comment by AliabbasMerchant** [comment]
I think yes.

File: grpc_server.go (hunk 1 above)
Comment (YourTechBud) [reply grpc_server.go]: Lol. Okay

**Pull request merged**